px/lg/templates/configmap-communities.yaml

```diff
@@ -0,0 +1,10 @@
+{{- $config_path := printf "%s%s.json" $.Values.lg_config_path "lg_px_communities_v2" -}}
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+    name: cfg-{{ .Values.region }}-pxrs-lg-communities
+data:
+#    "{{ $.Values.global.region }}-pxrs-lg-communities.cfg": |
+    "lg_px_communities_v2.json": |
+{{ $.Files.Get $config_path | indent 8}}
```

Comment:
It would be a good safe-guard to have a check that the file exists and not have helm secretly put an empty string there. Check the example here:
https://github.com/sapcc/helm-charts/blob/master/px/bird/templates/_configmap-bird.tpl#L14-L15